cluster/ubuntu/config-default.sh

```diff
@@ -37,6 +37,10 @@ export FLANNEL_NET=${FLANNEL_NET:-172.16.0.0/16}
 # Admission Controllers to invoke prior to persisting objects in cluster
 export ADMISSION_CONTROL=NamespaceLifecycle,LimitRanger,ServiceAccount,ResourceQuota,SecurityContextDeny
 
+# Path to the config file or directory of files of kubelet
+export KUBELET_CONFIG=${KUBELET_CONFIG:-""}
```

Comment:
My concern for passing this path w/o alerting the users could lead to potential conflict, let alone it's a recommended path in docker-k8s setup process.